### PR TITLE
Added changes from Discord Atlas Edit threads

### PR DESCRIPTION
**Pull request by GeoDash897**
I have made the following changes to the entries of the atlas:
- Update description of Emotiguy (Picardia) based on ali,kirb's thread (entry id: txfah1)
- Update boundary and description of INSA entry based on ⬛◽◾D◾◻⬛'s thread (entry id: 000371)
- Removed dupe Eren entry based on Novazert's thread (entry id: 000439)
- Updated Unturned entries based on GreatHeroJ's thread (entry ids: twr7jt, twwf4i, tx5zl9, tx4bi4)
- Fixed typo in Goku's Spirit Bomb entry (entry id: twmt33)

Note that while it says that I updated PearlescentMoon's entry, I decided to leave it after pulling from cleanup as I felt that the description given was better quality overall.

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -5818,9 +5817,9 @@
...
+{"id": "txfalv", "submitted_by": "SuperGameController", "name": "Skysail Insignia", "description": "A winged white sword.\n\nIt represents the Kingdom of Skysail in the realm of Inita, one of the many realms in the Countless Worlds.\n\nThe insignia of the winged sword mirrors the great military power of the Skysail Kingdom.\nNot only do they excel in military power, Skysail is the main trading hub within the realms, they control most of the trades between worlds and are responsible for exports of Nillium, a magical ore used in the hull of flying ships.\n\nThe game's discord can be found here > [discord.gg/CW](https://discord.gg/CW) <", "website": "https://www.roblox.com/games/6900861054/Countless-Worlds", "subreddit": "[/r/RobloxCountlessWorlds", "center": [297.5, 1508.5], "path": [[293.5, 1515.5], [300.5, 1515.5], [300.5, 1500.5], [293.5, 1500.5], [293.5, 1515.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:5820]: Fix the subreddit format here.

Comment (AnonymousRandomPerson) [inline web/atlas.json:5820]: Markdown format doesn't work here. Write out links in full without Markdown syntax.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Address comments.

File: web/atlas.json (hunk 1 above)
Comment (GeoDash897) [reply web/atlas.json:5820]: Gotcha, will do! Sorry about that

Comment (GeoDash897) [reply web/atlas.json:5820]: so it would just be https://discord.gg/CW then?

Comment (AnonymousRandomPerson) [reply web/atlas.json:5820]: Yes.

**Pull request merged**